sparse/compatibility.py

```diff
@@ -1,21 +1,20 @@
 # flake8: noqa F401
+import sys
 
-try:
-    from itertools import izip_longest as zip_longest
-except ImportError:
-    from itertools import zip_longest
+cur_version = sys.version_info
```

Comment:
I recomment avoiding the alias here and just using `sys.version_info` everywhere.  Otherwise when someone gets dropped into this file looking for a bug and they see `sys.version_info` rather than `cur_version` they will know immediately what it means rather than have to track down the reference.